nfs-server/sbin/fix-permissions

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+OWNER=65534
+GROUP=50
+BASEDIR='/nfs-volume/ninebox/kub'
+
+chown -R $OWNER:$GROUP $BASEDIR
+chmod -R u+rwX $BASEDIR                 # Give everything to owner.
+chmod -R g-rwx $BASEDIR                 # Take everything away from group to start.
+chmod -R g+rwX $BASEDIR                 # Give group rw of regular files, and rwx to directories.
+chmod -R o-rwx $BASEDIR                 # Remove all permissions from others.
+chmod -R o+rX $BASEDIR                  # Give others read on files, and rx on directories.
+# Hide all access from others for senssitive directories.
+chmod -R o-rwx $BASEDIR/*/tmp
```

Comment:
why do we keep putting in 2 temp dirs? pick on and stick with it

Reply:
I've tried to delete the one but they keep coming back. We have just the one thing using /app/resources/temp, and it's the authentication stuff.